### ISSUE TITLE
Split examples and user guide into separate sections

### ISSUE THREAD

**Issue by JoostBuitink** [Deltares/hydromt_wflow#250]
### HydroMT-Wflow version checks

- [X] I have checked that the issue still exists on the latest versions of the docs on `main` [here](https://github.com/Deltares/hydromt_wflow)

### Kind of issue

Docs are unclear

### Documentation problem

Currently, information and examples are mixed. This limits the current docs to include more information in a user-friendly way. For example, information in the pedo-transfer functions is lacking, information on the mapping tables is hidden through .csv files (and will be more hidden once https://github.com/Deltares/hydromt_wflow/pull/249 is merged). It would be nice if we have more information in the docs about the underlying assumptions and calculations that we do. 

The current docstrings would be one place to put this information, but I think it would make the code too messy (and perhaps not too easy to find). By splitting the examples and user guide into separate sections would give more space to include this information. This would also create space to put more information on the LAI pre-processing (https://github.com/Deltares/hydromt_wflow/issues/157) and the dependencies between the methods (https://github.com/Deltares/hydromt_wflow/issues/125).

**Comment by alimeshgi** [comment]
@JoostBuitink the structural split between examples and user guide was implemented in PR #600 , can I close this issue?

**Comment by JoostBuitink** [comment]
Yes I think so, although I still have some doubts whether the current locations of the examples is maybe to hidden (collapsible menu under "getting started"). Would it maybe make more sense to include a "examples" header/tab to the top bar (next to the "Getting started", "User guide", etc). Let's have a chat about it!

**Comment by alimeshgi** [comment]
Closing this issue. The examples/user guide split was implemented in PR #600. The minor follow-up about examples nav visibility can be tracked in a new issue if needed.
